src/dispatch/static/dispatch/src/components/FancyTooltip.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
```

Comment:
I think a more descriptive name might be: `HotKeyTooltip.vue`

Reply:
The hotkeys are optional, I agree `Fancy` is not a good name, and maybe it should just be called `Tooltip` (in contrast to `VTooltip` for vuetify.

Reply:
I renamed to `DMenu`/`DTooltip`  to follow Vuetify `VMenu`, `VTooltip` convention. The idea is we denoted `D` for Dispatch components, which are built on top of general concept Vuetify components.